libs/ui-toolkit/checkbox/src/lib/checkbox.component.ts

```diff
@@ -0,0 +1,125 @@
+import { Component, EventEmitter, Input, OnChanges, OnInit, Output, ViewEncapsulation } from '@angular/core';
+
+/**
+ * Interface for event emitted by CheckboxComponent
+ */
+export interface CheckboxEvent {
+  id: string | number;
+  value: boolean;
+}
+
+@Component({
+  selector: 'gor-checkbox',
+  templateUrl: './checkbox.component.html',
+  styleUrls: ['./checkbox.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class CheckboxComponent implements OnInit, OnChanges {
+  /**
+   * An id to identify the checkbox. Emitted together with the
+   * checkbox value when user clicks the checkbox.
+   * @default 0
+   */
+  @Input() public id: string | number = 0;
+
+  /**
+   * Checkbox label.
+   * @default label
+   */
+  @Input() public label = 'Label';
+
+  /**
+   * Label text transform.
+   * - up: uppercase
+   * - lo: lowercase
+   * - cp: capitalize
+   * - no: none
+   * @default up
+   */
+  @Input() public labelTransform: 'up' | 'lo' | 'cp' | 'no' = 'up';
+
+  /**
+   * Checkbox size.
+   * - sm: small (12px)
+   * - md: medium (14px)
+   * - lg: large (16px)
+   *
+   * @default 'sm'
+   */
+  @Input() public size: 'sm' | 'md' | 'lg' = 'sm';
```

Comment:
font size can be set by container, you can inspect in https://material.angular.io/components/checkbox/examples and change `mat-card-content` font-size to see changes